PER.md

```diff
@@ -0,0 +1,25 @@
+# PHP Evolving Recommendations
+
+A PHP Evolving Recommendations is a meta document accompanied by one or more artifacts that are set to evolve over time with multiple releases.
+Following the [PER Workflow Bylaw][workflow], when a PER has passed the Entrance Vote it will be listed here as "In Development"; once the first release of an artifact is tagged, it will be listed as "Released".
+
+As also described in the [Mission and Structure Bylaw][structure], the Editor of a working group working on a PER is essentially the lead contributor and writer, and they are supported by at least two voting members; the Sponsor is instead a Project Representative or Core Committee member that acts as a steward and helps pushing the proposal forward.
```

Comment:
```suggestion
As also described in the [Mission and Structure Bylaw][structure], the Editor of a working group working on a PER is essentially the lead contributor and writer, and they are supported by at least two voting members; the Sponsor, if present, is a Core Committee member that assists the Editor in keeping the evolution process on track.
```

Reply:
I agree with the change, but Project Reps can be sponsors, you left that out: https://github.com/php-fig/fig-standards/blob/803f9f6d94a22a1624ace8af38db0aadb34ed915/bylaws/001-mission-and-structure.md?plain=1#L13

Reply:
Project Reps can sponsor a project membership application.  They cannot sponsor a Working Group.

Reply:
Applied in [161d8a0](https://github.com/php-fig/fig-standards/pull/1281/commits/161d8a003ebf06dc387fe7e824be141881db5b9f).